R/MsBackend.R

```diff
@@ -298,6 +298,12 @@
 #'   spectra in the backend. Note that just writeable backends support this
 #'   method.
 #'
+#' - `reset` a backend (if supported). This method will be called on the backend
+#'   by the `reset,Spectra` method that is supposed to restore the data to its
+#'   original state (see `reset,Spectra` for more details). The function
+#'   returns the *resetted* backend. The default implementation for `MsBackend`
```

Comment:
I am not sure about *resetted*. IMHO it is *reset, reset, reset* (presence, past, perfect; similar to set, set, set). 
```suggestion
#'   returns the *reset* backend. The default implementation for `MsBackend`
```